themes/index.js

```diff
@@ -203,6 +203,12 @@ const themes = {
     text_color: "ebcfb2",
     bg_color: "373f51",
   },
+  indian: {
```

Comment:
Let's prefix the theme with "flag" 
`flag-india`

In future we can distinguish between flag themes then, so that all country flag themes would be scoped with one prefix.

Reply:
new color's:

flag_india: {
    title_color: "ff8f1c",
    icon_color: "250E62",
    text_color: "509E2F",
    bg_color: "ffffff",
  },